cookiecutter.json

```diff
@@ -20,11 +27,28 @@
     "R",
     "c++"
   ],
-  "framework": [
+  "gpu_usage": [
+    "yes",
+    "no"
+  ],
+  "CUDA_version": [
+    "9.0",
+    "10.0",
+    "10.1"
+  ],
+  "framework_if_using_python": [
+    "none",
     "pytorch",
     "tensorflow"
   ],
-  "use_gpu_for_computations":"no",
+  "use_pylint": [
```

Comment:
Are we using pylint or black and flake8? I think the two latter are better IMO but I might be wrong